x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_landing/create_integration_landing.tsx

```diff
@@ -36,100 +41,115 @@ const useAssistantCardCss = () => {
   `;
 };
 
-export const CreateIntegrationLanding = React.memo(() => {
+const IntegrationAssistantCard = React.memo(() => {
+  const { canExecuteConnectors } = useAuthorization();
```

Comment:
I extracted the Assistant's `EuiCard` in this separate component, which makes the file diff a bit confusing. 
The important changes are:
- Check of `canExecuteConnectors` for the Assistant button.
- Use of `<AuthorizationWrapper canCreateIntegrations>` wrapping the entire section body.